cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.TextField()
```

Comment:
In your `Movie` model, you've set the `description` field as `TextField`. However, if you know the maximum length of the description, it's more reasonable to use `CharField` instead. This is because `CharField` is meant for smaller amounts of text. If you don't expect the movie description to exceed a certain limit, consider changing `TextField` to `CharField` and set a `max_length`.